config-solaris.md

```diff
@@ -26,7 +26,7 @@ The property should consist of a comma-separated privilege set specification as
 ## <a name="configSolarisMaxShmMemory" />maxShmMemory
 The maximum amount of shared memory allowed for this application container.
 A scale (K, M, G, T) can be applied to the value for each of these numbers (for example, 1M is one megabyte).
-Mapped to `max-shm-memory` in [zonecfg(1M)][zonecfg.1m] man page.
+Mapped to `max-shm-memory` in [zonecfg(1M)][zonecfg.1m_2] man page.
```

Comment:
I got 404 error from this link, I think the right link is: https://docs.oracle.com/cd/E36784_01/html/E36871/zonecfg-1m.html

And I think it makes more sense to use alias `zonecfg.1m` for this link.